src/PerformanceTests/NServiceBus7/App.config

```diff
@@ -106,24 +89,4 @@
     </assemblyBinding>
     <gcServer enabled="false" />
   </runtime>
-  <system.net>
-    <settings>
-      <servicePointManager expect100Continue="false" useNagleAlgorithm="false" />
-    </settings>
-    <connectionManagement>
-      <add address="*" maxconnection="150" />
-    </connectionManagement>
-  </system.net>
-  <system.web>
-    <membership defaultProvider="ClientAuthenticationMembershipProvider">
```

Comment:
not sure what these would be required for?

Reply:
I'm not sure what the `system.web` stuff is for, I would remove it and I'll see if one of transports require it but I think not. Its different for the `system.net` section, that must remain.